enterprise/utils.py

```diff
@@ -49,19 +49,7 @@ def __init__(self, *args, **kwargs):
         Log a warning and initialize the exception.
         """
         LOGGER.warning('edx-enterprise unexpectedly failed as if not installed in an OpenEdX platform')
-        super(NotConnectedToEdX, self).__init__(*args, **kwargs)
-
-
-class NotConnectedToOpenEdX(Exception):
-    """
-    Exception to raise when we weren't able to import needed resources.
-
-    This is raised when we try to use the resources, rather than on
-    import, so that during testing we can load the module
-    and mock out the resources we need.
```

Comment:
To tell you the truth, this PR was picked up from @haikuginger, and this change was a part of it, so I decided to keep it going. It doesn't look like to me the changes are exactly that significant, it's just a squashing of two exception classes to maybe simplify what it is we work with.

I guess its original form could be considered a sort of minor tech debt, and it would have been better to eventually clean it up than to simply ignore it up until a dedicated PR. Since it's here and not really bothering anyone, I suppose we can continue with it, is that okay? If not, I'll open a dedicated PR to do it but will take more work. Either that or I can separate the commits for this change and for the rest in this PR.